MerchantSDK/GiniMerchantSDK/Sources/GiniMerchantSDK/Core/PaymentReviewContainerView.swift

```diff
@@ -0,0 +1,694 @@
+        stackView.distribution = .fill
+        return stackView
+    }()
+
+    private lazy var ibanAmountHorizontalStackView: UIStackView = {
+        let stackView = EmptyStackView(orientation: .horizontal)
+        stackView.distribution = .fill
+        stackView.spacing = Constants.stackViewSpacing
+        return stackView
+    }()
+
+    private lazy var ibanTextFieldView: TextFieldWithLabelView = {
+        let textFieldView = TextFieldWithLabelView()
+        textFieldView.tag = TextFieldType.ibanFieldTag.rawValue
+        return textFieldView
+    }()
+
+    private lazy var amountTextFieldView: TextFieldWithLabelView = {
+        let textFieldView = TextFieldWithLabelView()
+        textFieldView.tag = TextFieldType.amountFieldTag.rawValue
+        return textFieldView
+    }()
+
+    private lazy var ibanAmountErrorsHorizontalStackView: UIStackView = {
+        let stackView = EmptyStackView(orientation: .horizontal)
+        stackView.distribution = .fill
+        return stackView
+    }()
+
+    private lazy var ibanErrorLabel: UILabel = {
+        let label = UILabel()
+        return label
+    }()
+
+    private lazy var amountErrorLabel: UILabel = {
+        let label = UILabel()
+        return label
+    }()
+
+    private lazy var usageStackView: UIStackView = {
+        let stackView = EmptyStackView(orientation: .vertical)
+        stackView.distribution = .fill
+        return stackView
+    }()
+
+    private lazy var usageTextFieldView: TextFieldWithLabelView = {
+        let textFieldView = TextFieldWithLabelView()
+        textFieldView.tag = TextFieldType.usageFieldTag.rawValue
+        return textFieldView
+    }()
+
+    private lazy var usageErrorLabel: UILabel = {
+        let label = UILabel()
+        return label
+    }()
+
+    private lazy var buttonsView: UIView = {
+        EmptyView()
+    }()
+
+    private lazy var buttonsStackView: UIStackView = {
+        EmptyStackView(orientation: .horizontal)
+    }()
+
+    private lazy var payInvoiceButton: PaymentPrimaryButton = {
+        let button = PaymentPrimaryButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.frame = CGRect(x: 0, y: 0, width: .greatestFiniteMagnitude, height: Constants.buttonViewHeight)
+        return button
+    }()
+
+    private lazy var bottomView: UIView = {
+        EmptyView()
+    }()
+
+    private lazy var bottomStackView: UIStackView = {
+        EmptyStackView(orientation: .horizontal)
+    }()
+
+    private lazy var poweredByGiniView: PoweredByGiniView = {
+        let view = PoweredByGiniView()
+        view.viewModel = PoweredByGiniViewModel()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    var amountToPay = Price(value: 0, currencyCode: "€")
+    var lastValidatedIBAN = ""
+
+    var paymentInputFields: [TextFieldWithLabelView] = []
+    var paymentInputFieldsErrorLabels: [UILabel] = []
```

Comment:
[Clarification]: do the variables below should be private as well?
     var amountToPay 
     var lastValidatedIBAN 
     var paymentInputFields: 
     var paymentInputFieldsErrorLabels